backend/dataall/modules/datasets/api/dataset/resolvers.py

```diff
@@ -24,7 +27,7 @@ def create_dataset(context: Context, source, input=None):
 
 
 def import_dataset(context: Context, source, input=None):
-    RequestValidator.validate_import_request(input)
+    RequestValidator.validate_import_request(input, context)
```

Comment:
do you need to pass context here anymore?

Reply:
Not needed. I think refactoring missed this one. Thanks for catching. Will update in the next update